hack/chart-verifier-podman.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# KUBECONFIG is either the value the user has configured in the
+# environment or the default location `kubectl` uses.
+KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/config}"
+
+# CHART_VERIFIER_IMAGE is the location of the image to use to verify
+# the chart.
+CHART_VERIFIER_IMAGE="${CHART_VERIFIER_IMAGE:-quay.io/redhat-certification/chart-verifier:main}"
+
+# PODMAN contains the path for the `podman` program.
+PODMAN="${PODMAN:-/usr/bin/podman}"
+
+"${PODMAN}" run --rm -i                      \
+            --log-level="error"              \
+            --systemd="false"                \
+            --cgroup-manager="cgroupfs"      \
+            --cgroups="disabled"             \
+            --events-backend="none"          \
+            --log-driver="none"              \
```

Comment:
are these options really needed? I tried it without them and it worked.

Reply:
Since I'm testing in a more restrictive environment (Ubuntu 20.04 on WSL2) those are required to run `podman`, which are assumptions Docker doesn't have (for example, that the host system has "journald" active.